data/json/items/gun/762.json

```diff
@@ -115,5 +114,79 @@
     ],
     "magazines": [ [ "762", [ "762x39_clip" ] ] ],
     "flags": [ "RELOAD_ONE", "NEVER_JAMS" ]
+  },
+  {
+    "id": "aksemi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "AK-47" },
```

Comment:
Do we want to go with the "familiarity" way? I.e. most players recognize what AK-47 is.
Or with the "actuality" way? I.e. there is **no** IRL rifles named AK-47. The real rifles named simply "AK", without any numbers. Rifles named "AK-47" is a pilot batch. They were later officially taken into operation as "7,62mm Avtomat Kalashnikova (AK)".

Reply:
Generally, the AKM pattern rifles have 'AK' and '47' in their name. Given that the PC is (usually) an American, not a Cyrillic reading com-bloc aficionado, What retailers and enthusiasts refer to this pattern of rifle as is probably the right name. [For example](https://www.classicfirearms.com/long-guns/ak-47-for-sale/7.62x39/), [and here](https://www.atlanticfirearms.com/taxons/ak47-for-sale).

Reply:
My thoughts:

AK-47: colloquial and common marketing name but technically a misnomer.  Probably my choice of the 4, but I'm not a huge fan of any of them.
AK: too short of a name at 2 characters.
AK Sporter: a possible alternative, but to me it invokes images of thumbhole stocks and unconverted Saigas.
AK Semi-auto: a bit too janky in my opinion.

I'm going to wait until sometime tonight to let this discussion and other possible issues ferment before I make yet another commit.

Reply:
In the absence of any other input, I think I'm going to keep it as the AK-47.  Yes, I am aware of the history of the Kalashnikov and that no (actually used) rifle is really called the AK-47, but it is hard to neatly fit all of the different civilian models under one catch-all name like the AR-15 can.  We *could* make the WASR-10, the Zastava N-PAP, the Arsenal SGL-21, the Century C39V2, ad astra, to be technically correct, but that would just be useless bloat the same way adding Colt, BCM, Daniel Defense, Del-ton AR-15s would.  While realism and technical correctness seem to be appreciated where possible, this is one of those times where splitting hairs isn't going to get anywhere useful it seems.  The name might yank the chains of some "gun guys" playing the game, but we need to remember that we are in the minority, and that the majority of players I think will be fine with it so long as they read the description/stats and understand it isn't an automatic assault rifle.